synfig-studio/src/gui/app.cpp

```diff
@@ -1823,7 +1825,7 @@ App::~App()
 
 	main_window->hide();
 
-	delete main_window;
+	// delete main_window;
```

Comment:
Looks like we'll need to refactor this part in the future.

Reply:
If we move this part to the end of the method, then it works:
```
	process_all_events();
	delete main_window;
```

Reply:
Great!
Done!